lib/config.js

```diff
@@ -5,6 +5,11 @@ const EndpointAddress = {
   development: 'api.sandbox.push.apple.com',
 };
 
+const ManageBroadcastEndpointAddress = {
+  production: 'api-manage-broadcast.push.apple.com',
```

Comment:
A new endpoint is required for managing channels discussed here: https://developer.apple.com/documentation/usernotifications/sending-channel-management-requests-to-apns